React/Views/RCTMapManager.m

```diff
@@ -73,6 +74,27 @@ - (void)mapView:(RCTMap *)mapView didSelectAnnotationView:(MKAnnotationView *)vi
   }
 }
 
+- (void) mapView:(RCTMap *)mapView annotationView:(MKAnnotationView *)view didChangeDragState:(MKAnnotationViewDragState)newState fromOldState:(MKAnnotationViewDragState)oldState
+{
+  if ([view.annotation isKindOfClass:[RCTPointAnnotation class]]) {
+    
+    RCTPointAnnotation *annotation = (RCTPointAnnotation *)view.annotation;
+    
+    if (mapView.onAnnotationDragChange) {
+      mapView.onAnnotationDragChange(@{
+        @"state": @(newState),
```

Comment:
The state is a enum that indicate state of the drag "dragging", "canceled" , "drop", etc. I'll get right on this.